gobblin-runtime/src/main/java/org/apache/gobblin/service/monitoring/JobStatus.java

```diff
@@ -40,7 +40,8 @@
   private final long startTime;
   private final long endTime;
   @Setter
-  private String message;
+  private String metrics;
```

Comment:
I think the Setter is needed for the message field too.

Reply:
Actually I added the setter in https://github.com/apache/incubator-gobblin/pull/3076 to add metrics in message field, so it can be set in internal projects. But now I am adding a separate metrics field for that purpose, which is to be set by custom job status retriever.